libvast/src/taxonomies.cpp

```diff
@@ -161,7 +156,7 @@ resolve_concepts(const concepts_type& concepts, const expression& e,
       // All fields that the concept is resolve to either directly or indirectly
       // through referenced concepts.
       detail::stable_set<std::string> target_fields;
-      auto handle_def = [&](const concepts_type::definition& def) {
+      auto handle_def = [&](const concept_& def) {
```

Comment:
At the call site, the semantics of this function name are almost impossible to infer. Maybe you can use a verb that is a more specific to the algorithm than `handle`?

Reply:
The call sites are directly below in the same section of the code, so I didn't think this would be an issue. The only alternative that comes to my mind is `load`, but I'm not sure that's an improvement.

Reply:
I find `load` is much better, actually, wich `load_definition` being my favorite.

(But no need to spend more on this detail; take it or leave it. Marking this as resolved.)